### ISSUE TITLE
-50% energy reduction, no cool down buff not working right?

### ISSUE THREAD

**Issue by SuplexCity25284** [PavelZinchenko/event-horizon-frontier-builds#338]
I tried using it on my current player ship, Scarab, didn't work.  Had no energy gain, so ship was dead in the water, no power.  Put it in a merc ship, works fine.  Flew it, the merc ship, in exploration, it works as the description on the buff says.  

What's the problem?

**Comment by SuplexCity25284** [comment]
Upon further inspection it only seems to work on Space Invader and chronos, and to lesser extent on Oracle and Valkyrie

**Comment by Absolutan115** [comment]
Example
Power usage 100
Energy regen with perk 200/2=100
Result 0 Energy. All modul shutdown

**Comment by SuplexCity25284** [comment]
Yes.  But when put on Space Invader, Chronos, even with power regen dropped, the power does recharge.  On the ships it doesn't work on, instead of dropping regeration by 50%, it drops it to 0 power, or even -1.  

My point is on some ships, while it says it drops power regen by 50%, it drops it by 100%

**Comment by SuplexCity25284** [comment]
This is what I mean: 
![Uploading Screenshot_20240719_131435_Event Horizon - Frontier.jpg…]()
That is before the change.

![Uploading Screenshot_20240719_131212_Event Horizon - Frontier.jpg…]()
This is after.  

I get the debuff on my stored energy.  But on other ships it doesn't drop the produced energy nearly as much.

**Comment by SuplexCity25284** [comment]
Like I said, if I look at Space Invader, which a merc uses, and I've installed this mod.  

If you look at my screenshots: 

![Uploading Screenshot_20240719_131942_Event Horizon - Frontier.jpg…]()
![Uploading Screenshot_20240719_132206_Event Horizon - Frontier.jpg…]()

It halves stored power, but cuts production by approximately 5 times.  How is that 50% but 0 cool down?

Using the math from Space Invader, Scarab should have about 69 energy production, but has -53

**Comment by SuplexCity25284** [comment]
This screenshots never loaded for some reason

**Comment by SuplexCity25284** [comment]
Here: 
![Uploading Screenshot_20240719_131212_Event Horizon - Frontier.jpg…]()
![Uploading Screenshot_20240719_131435_Event Horizon - Frontier.jpg…]()
![Uploading Screenshot_20240719_131942_Event Horizon - Frontier.jpg…]()
![Uploading Screenshot_20240719_132206_Event Horizon - Frontier.jpg…]()

**Comment by magne00** [comment]
Cannot see the screenshots so i can't tell. But my guess is, you have modules that consume alot of energy. Module's energy cost are not affected by the perk.

**Comment by SuplexCity25284** [comment]
The screenshots simply won't load, not on my phones lte, or on the wifi at work

**Comment by SuplexCity25284** [comment]
And rhe numbers I mention are including the items on the ship at the time, so any load on the reactors is already there.

**Comment by SuplexCity25284** [comment]
> Cannot see the screenshots so i can't tell. But my guess is, you have modules that consume alot of energy. Module's energy cost are not affected by the perk.

That's probably true, but I left them installed while I added and removed the perk, so it shouldn't affect the outcome

**Comment by SuplexCity25284** [comment]
> Cannot see the screenshots so i can't tell. But my guess is, you have modules that consume alot of energy. Module's energy cost are not affected by the perk.



That's probably true, but I left them installed while I added and removed the perk, so it shouldn't affect the outcome
